[2025-03-14T19:35:01.930+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-14T19:35:02.022+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: open_weather_etl.transform scheduled__2025-03-14T19:30:00+00:00 [queued]>
[2025-03-14T19:35:02.026+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: open_weather_etl.transform scheduled__2025-03-14T19:30:00+00:00 [queued]>
[2025-03-14T19:35:02.026+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-14T19:35:02.032+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): transform> on 2025-03-14 19:30:00+00:00
[2025-03-14T19:35:02.038+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=385) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-14T19:35:02.039+0000] {standard_task_runner.py:72} INFO - Started process 386 to run task
[2025-03-14T19:35:02.040+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'open_weather_etl', 'transform', 'scheduled__2025-03-14T19:30:00+00:00', '--job-id', '147', '--raw', '--subdir', 'DAGS_FOLDER/open_weather_dag.py', '--cfg-path', '/tmp/tmpgn80la07']
[2025-03-14T19:35:02.040+0000] {standard_task_runner.py:105} INFO - Job 147: Subtask transform
[2025-03-14T19:35:02.062+0000] {task_command.py:467} INFO - Running <TaskInstance: open_weather_etl.transform scheduled__2025-03-14T19:30:00+00:00 [running]> on host c20e6545b232
[2025-03-14T19:35:02.102+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='lherrera5034@gmail.com' AIRFLOW_CTX_DAG_OWNER='Airflow' AIRFLOW_CTX_DAG_ID='open_weather_etl' AIRFLOW_CTX_TASK_ID='transform' AIRFLOW_CTX_EXECUTION_DATE='2025-03-14T19:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-14T19:30:00+00:00'
[2025-03-14T19:35:02.102+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-14T19:35:02.108+0000] {logging_mixin.py:190} INFO - Data trasnformed
[2025-03-14T19:35:02.109+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-14T19:35:02.114+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-14T19:35:02.114+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=open_weather_etl, task_id=transform, run_id=scheduled__2025-03-14T19:30:00+00:00, execution_date=20250314T193000, start_date=20250314T193502, end_date=20250314T193502
[2025-03-14T19:35:02.138+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-14T19:35:02.148+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-14T19:35:02.149+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
